src/main/java/com/eta/houzezbackend/controller/AgentController.java

```diff
@@ -29,8 +29,8 @@ public AgentGetDto getAgent(@PathVariable Long id) {
     }
 
 
-    @PatchMapping("/verification")
-    public Agent activeAgent(@RequestParam String token){
+    @PatchMapping("/verification/{id}")
```

Comment:
Think about the meaning behind restful API, do we need "verification" here? ` @PatchMapping("/{id}")`